flexget/plugins/list/series_list.py

```diff
@@ -0,0 +1,283 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+from collections import MutableSet
+from datetime import datetime
+
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+from sqlalchemy import Column, Unicode, Integer, ForeignKey, Boolean, DateTime, String, func
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.elements import and_
+
+from flexget.plugins.filter.series import FilterSeriesBase
+from flexget import plugin
+from flexget.db_schema import versioned_base
+from flexget.utils.database import json_synonym, with_session
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.tools import split_title_year
+
+log = logging.getLogger('series_list')
+Base = versioned_base('series_list', 0)
+
+SUPPORTED_IDS = FilterSeriesBase().supported_ids
+SETTINGS_SCHEMA = FilterSeriesBase().settings_schema
+SERIES_ATTRIBUTES = SETTINGS_SCHEMA['properties']
+
+
+class SeriesListList(Base):
+    __tablename__ = 'series_list_lists'
+    id = Column(Integer, primary_key=True)
+    name = Column(Unicode, unique=True)
+    added = Column(DateTime, default=datetime.now)
+    series = relationship('SeriesListSeries', backref='list', cascade='all, delete, delete-orphan', lazy='dynamic')
+
+    def __repr__(self):
+        return '<SeriesListList name=%d>' % (self.id)
+
+    def to_dict(self):
+        return {
+            'id': self.id,
+            'name': self.name,
+            'added_on': self.added
+        }
+
+
+class SeriesListSeries(Base):
+    __tablename__ = 'series_list_series'
+
+    id = Column(Integer, primary_key=True)
+    added = Column(DateTime, default=datetime.now)
+    title = Column(Unicode)
+
+    # Internal series attributes
+    set = Column(Unicode)
+    path = Column(Unicode)
+    _alternate_name = Column('alternate_name', Unicode)
+    alternate_name = json_synonym('_alternate_name')
```

Comment:
The idea was to avoid defining numerous tables while still easily maintaining list structure. Any ideas how to achieve this otherwise?